src/loader.js

```diff
@@ -1,3 +1,6 @@
+#!/usr/bin/env node
```

Comment:
It doesn't look like this file does anything meaningful when used as the entrypoint. Do we need this change here?